tests/torchtune/utils/test_distributed.py

```diff
@@ -397,6 +399,14 @@ def test_lora_state_dict(self):
 
     @gpu_test(gpu_count=2)
     def test_qlora_state_dict(self):
+        self.run_subtests(
+            {
+                "enable_activation_checkpointing": [False, True],
+            },
+            self._test_qlora_state_dict,
+        )
```

Comment:
nit: can just use `@pytest.mark.parametrize` for this? (Unless there's something about `FSDPTest` that precludes that, if so feel free to ignore this)